agent/packaging/ubuntu_xenial/kontena-agent/etc/kontena-agent.env

```diff
@@ -1,5 +1,5 @@
 # Set URL of your kontena server
-#KONTENA_URI=wss://localhost:8443
+#KONTENA_URI=ws://127.0.0.1:8080
```

Comment:
That's what's in the debconf... I'll change this to localhost, seems like docker-proxy listens on both IPv4 and IPv6:

```
LISTEN     0      128                                       :::8443                                                  :::*                   users:(("docker-proxy",pid=5114,fd=4))
LISTEN     0      128                                       :::8080                                                  :::*                   users:(("docker-proxy",pid=5147,fd=4))
```